clients/src/main/java/org/apache/kafka/clients/consumer/internals/FetchBuffer.java

```diff
@@ -185,6 +188,16 @@ void awaitNotEmpty(Timer timer) {
         }
     }
 
+    void wakeup() {
```

Comment:
nit: `wakeup` vs. `wakeUp` capitalization

Reply:
I used the same format as on the `Consumer` interface. Same for my recent change to `wokenup`.